examples/with_winit/src/lib.rs

```diff
@@ -268,6 +268,7 @@ fn run(
                 images: &mut images,
                 resolution: None,
                 base_color: None,
+                interactive: false,
```

Comment:
… and true here? That’s what I would expect semantically and seems to align with the actual use of the value but I may be missing something.

Reply:
Thanks! :sweat_smile: